src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -48,7 +47,7 @@
               <span class="fa fa-search" aria-hidden="true"></span>
             </button>
           </div>
-          <kbn-typeahead-items></kbn-typeahead-items>
+          <kbn-typeahead-items><query-help></query-help></kbn-typeahead-items>
```

Comment:
I internally debated over asking for this because it's kind of nitpicky, but I might as well... can we format this to use indentation to reflect the nested natured of these elements?

```html
<kbn-typeahead-items>
  <query-help></query-help>
</kbn-typeahead-items>
```